src/test/java/teammates/test/pageobjects/AdminEmailPage.java

```diff
@@ -17,6 +17,7 @@ protected boolean containsExpectedPageContents() {
 
     public void inputRecipient(String recipient) {
         WebElement recipientBox = this.getRecipientBox();
+        recipientBox.clear();
```

Comment:
Understood. Recommend creating a new method for doing this, like the `clearSubjectBox()` method. The input* methods all do not clear their element, so changing this one to clear current input causes surprising behaviour.